src/main/java/site/coach_coach/coach_coach_server/action/service/ActionService.java

```diff
@@ -23,12 +24,26 @@ public class ActionService {
 	@Transactional
 	public Long createAction(Long routineId, Long categoryId, Long userIdByJwt,
 		CreateActionRequest createActionRequest) {
-		routineService.validateAccessToRoutine(routineId, userIdByJwt);
+		routineService.validateBeforeModifyRoutineDetail(routineId, userIdByJwt);
 		Category category = categoryRepository.findByCategoryIdAndRoutine_RoutineId(categoryId, routineId)
 			.orElseThrow(() -> new NotFoundCategoryException(ErrorMessage.NOT_FOUND_CATEGORY));
 
 		Action action = Action.of(createActionRequest, category);
 
 		return actionRepository.save(action).getActionId();
 	}
+
+	@Transactional
+	public void deleteAction(Long routineId, Long categoryId, Long actionId, Long userIdByJwt) {
+		routineService.validateBeforeModifyRoutineDetail(routineId, userIdByJwt);
+		categoryRepository.findByCategoryIdAndRoutine_RoutineId(categoryId, routineId)
+			.orElseThrow(() -> new NotFoundCategoryException(ErrorMessage.NOT_FOUND_CATEGORY));
+
+		Boolean isExistsAction = actionRepository.existsById(actionId);
+		if (isExistsAction) {
```

Comment:
```suggestion
		if (actionRepository.existsById(actionId)) {
```
변수 선언없이 이렇게 바로 할 수도 있겠네요~! 하지만 취향차이인 것 같긴해요 저는 변수만들기 싫어서 간단하면 바로 이렇게 하거든요

Reply:
저는 if분기문의 조건에 길게 넣는걸 싫어하는데
`actionRepository.existsById(actionId)` 요정도는 넣어도 괜찮을거 같네요 ㅎㅎ
반영할게요!